tests/features/import-explorer.feature

```diff
@@ -0,0 +1,135 @@
+Feature: Importer Explorer - View and configure importers on importers details page
+  Background: As a Devsecops I want to view and configure all of the Importers currently configured including default Importers
```

Comment:
The condition background are similar to "BeforeTest" it would be executed for all the scenarios under feature file. We can have the Given statements into this.
```suggestion
Feature: Importer Explorer - View and configure importers on importers details page
  Background: 
      Given the user navigates to and logs into the TPA application
  
```